From 2344b8e73a6e22da55a340e12de559e781c2dff4 Mon Sep 17 00:00:00 2001
From: Dinesh kumar <dineshkumar.varadarajan@adlinktech.com>
Date: Thu, 4 Apr 2019 15:49:50 +0530
Subject: [PATCH 13/47] Clarify kernel message indicating the CPU variant.

The kernel code mostly talks about i.MX6 "Quad" and "DualLite", but they
actually mean "Quad/Dual" and "DualLite/Solo". Having received complaints
about the wrong CPU being detected we better clarify this a bit.

Signed-off-by: Dinesh kumar <dineshkumar.varadarajan@adlinktech.com>
---
 arch/arm/mach-imx/mach-imx6q.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/mach-imx6q.c b/arch/arm/mach-imx/mach-imx6q.c
index 7b2462f..41287c6 100644
--- a/arch/arm/mach-imx/mach-imx6q.c
+++ b/arch/arm/mach-imx/mach-imx6q.c
@@ -322,7 +322,7 @@ static void __init imx6q_init_machine(void)
 	if (cpu_is_imx6q() && imx_get_soc_revision() >= IMX_CHIP_REVISION_2_0)
 		imx_print_silicon_rev("i.MX6QP", IMX_CHIP_REVISION_1_0);
 	else
-		imx_print_silicon_rev(cpu_is_imx6dl() ? "i.MX6DL" : "i.MX6Q",
+		imx_print_silicon_rev(cpu_is_imx6dl() ? "i.MX6DL/S" : "i.MX6Q/D",
 				imx_get_soc_revision());
 
 	parent = imx_soc_device_init();
-- 
2.7.4

